voltixApp/VoltixApp/Services/BalanceService.swift

```diff
@@ -60,10 +54,7 @@ class BalanceService {
         case .gaiaChain:
             let atomBalance =  try await gaia.fetchBalances(address: coin.address)
             var balanceUSD: String = .empty
-            if let priceRateUsd = await CryptoPriceService.shared.cryptoPrices?.prices[coin.priceProviderId]?["usd"] {
-                balanceUSD = atomBalance.atomBalanceInUSD(usdPrice: priceRateUsd) ?? "US$ 0,00"
-                coin.priceRate = priceRateUsd
-            }
+            balanceUSD = atomBalance.atomBalanceInUSD(usdPrice: coin.priceRate) ?? "$ 0,00"
```

Comment:
can we please stop calling it `atomBalanceInUSD` ? and `usdPrice` should be just `price`